standard/expressions.md

```diff
@@ -4507,7 +4533,7 @@ Anonymous functions in an argument list participate in type inference and overlo
 >
 > <!-- Example: {template:"standalone-lib", name:"OverloadResolution1"} -->
 > ```csharp
-> class ItemList<T> : List<T>
+> class ItemList<T> : System.Collections.Generic.List<T>
```

Comment:
Do we need this qualification any more, with the addition of the using directives?

Reply:
agreed; removed